charts/aws-efs-csi-driver/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Helm chart
+
+# v2.0.0
+
+## Notable changes
+
+* Multiple breaking changes in values file
+* Support only Kubernetes `>=1.18` with `CSIDriver` API version `storage.k8s.io/v1`
+
+## New features
```

Comment:
Can we have a specific section for "Breaking Changes"? Ideally we don't want the users to go read the template and figure out every single change they need to make.

Reply:
Tried to explain breaking changes in b1c989de876b5cc719153f51dbe3a831a0a731ed